tests/core/performance/test_perf_tests.py

```diff
@@ -0,0 +1,28 @@
+"""
+This files tests the performance of the test runner (openfisca-run-test).
+It uses the existing tests from openfisca-france because it is the largest set we currently have.
+
+"""
+
+import os
+import time
+import pkg_resources
+
+from openfisca_core.tools.test_runner import run_tests
+from openfisca_france import CountryTaxBenefitSystem
+
+start_time = time.time()
+tbs = CountryTaxBenefitSystem()
+time_spent = time.time() - start_time
+print("Generate TBS: --- %s seconds ---" % (time_spent))
```

Comment:
What about 
```py
print("Generate Tax Benefit System: --- {}s seconds ---".format(time_spent))
```
?
(`format` is the standard way to interpolate variables in strings since `2.6` 🙂 )